pkg/querysharding/analyzer_test.go

```diff
@@ -137,6 +137,11 @@ sum by (container) (
 			expression:     "increase(sum(http_requests_total) by (pod, cluster) [1h:1m])",
 			shardingLabels: []string{"cluster", "pod"},
 		},
+		{
+			name:           "ignore vector matching with 2 aggregations",
+			expression:     `sum(rate(node_cpu_seconds_total[3h]offset 1m)) by (cluster_id, mode) / ignoring(mode) group_left sum(rate(node_cpu_seconds_total[3h]offset 1m)) by (cluster_id)`,
```

Comment:
It might be better to remove parts that are not necessary, like `offset`, so that it's more clear what the case is testing.

Reply:
Updated. PTAL